externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/operation/SparkOperationSuite.scala

```diff
@@ -137,9 +137,6 @@ class SparkOperationSuite extends WithSparkSQLEngine {
         assert(pos === 18, "all columns should have been verified")
       }
 
-      val e = intercept[HiveSQLException](metaData.getColumns(null, "*", null, null))
```

Comment:
shall we add a positive test here instead?

Reply:
done